### PR TITLE
spelling issues fixed.

### PR DESCRIPTION
**Pull request by sujith-mn**

### CODE REVIEW

File: documentation/advanced-tooling-generic.asciidoc [hunk 1]
```diff
@@ -15,4 +15,4 @@ To draw diagrams for your project or for blueprints in devonfw, we recommend the
 
 == Browser Plugins
 There are tons of helpful browser plugins out there and it might be a matter of personal taste what you like to have installed. However, as we are heavily using github we want to promote https://github.com/buunguyen/octotree#octotree[octotree].
-In case you also work with ZenHub you might want to install the https://www.zenhub.com/extension[Zenhub Browser Extension].
+In case you also work with `ZenHub` you might want to install the https://www.zenhub.com/extension[Zenhub Browser Extension].
```
Comment (ssarmokadam) [inline documentation/advanced-tooling-generic.asciidoc:18]: Add AenHub to dict

File: documentation/eclipse.asciidoc [hunk 2]
```diff
@@ -31,7 +31,7 @@ There are variables that can be used for Eclipse. These are explained by the fol
 |=======================
 |*Variable*|*Default-Value*|*Meaning*
 |*`ECLIPSE_VERSION`*|`2020-12`|The version of the tool Eclipse to install and use.
-|*`ECLIPSE_EDITION_TYPE`*|`java`|The edition of the tool Eclipse to install and use. You can choose between 'java' for standard edition or 'jee' for enterprise edition.
+|*`ECLIPSE_EDITION_TYPE`*|`java`|The edition of the tool Eclipse to install and use. You can choose between 'java' for standard edition or `jee` for enterprise edition.
```
Comment (ssarmokadam) [inline documentation/eclipse.asciidoc:34]: java->Java add to dict
jee->JEE add to dict

File: documentation/features.asciidoc [hunk 3]
```diff
@@ -12,11 +12,11 @@ Set up your IDE within minutes tailored for the requirements of your project.
...
 * *Configurable* +
 You can change the link:configuration.asciidoc[configuration] depending on your needs. Furthermore, the link:settings.asciidoc[settings] contain configuration templates for the different tools (see link:configurator.asciidoc[configurator]).
 * *Maintainable* +
-For your project you should copy these link:settings.asciidoc[settings] to an own `git` repository that can be maintained and updated to manage the tool configurations during the project lifecycle. If you use github or gitlab every developer can easily suggest changes and improvements to these link:settings.asciidoc[settings] via pull/merge requests, which is easier to manage with big teams.
+For your project you should copy these link:settings.asciidoc[settings] to an own `git` repository that can be maintained and updated to manage the tool configurations during the project lifecycle. If you use `github` or `gitlab` every developer can easily suggest changes and improvements to these link:settings.asciidoc[settings] via pull/merge requests, which is easier to manage with big teams.
```
Comment (ssarmokadam) [inline documentation/features.asciidoc:19]: git ,github,gitlab ->dict

File: documentation/features.asciidoc [hunk 4]
```diff
@@ -95,9 +95,9 @@ Now you might ask:
 * But I use Windows/Linux/MacOS/… - it works on all platforms!
 * But how about Windows CMD or Power-Shell? - it works!
 * But what if I use cygwin or git-bash on windows? - it works!
-* But I love to use ConEmu or Commander - it works with full integration!
-* How about MacOS Terminal or iTerm2? - it works with full integration!
-* But I use zsh  - it works!
+* But I love to use `ConEmu` or `Commander` - it works with full integration!
+* How about `MacOS Terminal` or `iTerm2`? - it works with full integration!
```
Comment (ssarmokadam) [inline documentation/features.asciidoc:99]: MacOS ->dict?

File: documentation/node.asciidoc [hunk 5]
```diff
@@ -10,10 +10,10 @@ The arguments (`devon node «args»`) are explained by the following table:
...
 |=======================
 |*Argument(s)*   |*Meaning*
-|`setup`         |setup node.js (install and verify), link:configuration.asciidoc[configurable] via `NODE_VERSION`
+|`setup`         |setup `node.js` (install and verify), link:configuration.asciidoc[configurable] via `NODE_VERSION`
 |`create «name» [«args»]` | create a new devon4node application `(same as devon4node new)`
 |`generate «s» [«args»]`    | generate devon4node components using the schematic «s» `(same as devon4node generate)`
 |`db «c» [«args»]`          | execute a TypeORM command «c» `(same as devon4node db)`
 |`cicd «args»`              |generate cicd files for the current devon4node project
-|`«args»`                   | call NodeJS with the specified arguments
+|`«args»`                   | call `NodeJS` with the specified arguments
```
Comment (ssarmokadam) [inline documentation/node.asciidoc:18]: NodeJS->dict?

File: documentation/scripts.asciidoc [hunk 6]
```diff
@@ -40,7 +40,7 @@ This directory is the heart of the `devonfw-ide` and contains the required link:
 
 The https://github.com/devonfw/ide/tree/master/scripts/src/main/resources/scripts/command[command] folder contains the link:cli.asciidoc#commandlets[commandlets].
 The https://github.com/devonfw/ide/tree/master/scripts/src/main/resources/scripts/devon[devon] script is the key link:cli.asciidoc[command line interface] for `devonfw-ide`.
-There is also https://github.com/devonfw/ide/tree/master/scripts/src/main/resources/scripts/devon.bat[devon.bat] that can be used in CMD or PowerShell.
+There is also https://github.com/devonfw/ide/tree/master/scripts/src/main/resources/scripts/devon.bat[devon.bat] that can be used in `CMD` or `PowerShell`.
```
Comment (ssarmokadam) [inline documentation/scripts.asciidoc:43]: cmd and powershell ->dict?

**Review by ssarmokadam — CHANGES_REQUESTED** [review]

File: documentation/features.asciidoc (hunk 3 above)
Comment (SchettlerKoehler) [reply documentation/features.asciidoc:19]: Please check what is the correct spelling for all three e.g. is it github or GitHub or Github,...?

File: .spellcheck.yml [hunk 7]
```diff
@@ -59,5 +71,5 @@ matrix:
       context_visible_first: true
       delimiters:
       # Ignore multiline content between [source, *] and blank line
-      - open: '\[source,.*\]'
-        close: '^\s*$' 
+      - open: '\[source, .*\]\r?\n\s*----'
+        close: '[^-]*'
```
Comment (ssarmokadam) [inline .spellcheck.yml:75]: Correct regular expression is

```
- pyspelling.filters.context:
      context_visible_first: true
      delimiters:
      # Ignore multiline content between [source, *] and blank line
      - open: '\[source,.*\]\r?\n\s*----'
        close: '\s*----'
```

File: documentation/features.asciidoc (hunk 4 above)
Comment (SchettlerKoehler) [reply documentation/features.asciidoc:99]: All names should be in the dict

File: documentation/features.asciidoc [hunk 8]
```diff
@@ -95,9 +95,9 @@ Now you might ask:
...
+* But I use `zsh`  - it works!
```
Comment (SchettlerKoehler) [inline documentation/features.asciidoc:100]: zsh is also a name, I think. Please check this.

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: .spellcheck.yml (hunk 7 above)
Comment (sujith-mn) [reply .spellcheck.yml]: Done

File: documentation/advanced-tooling-windows.asciidoc [hunk 9]
```diff
@@ -81,7 +81,7 @@ See also https://www.howtogeek.com/128680/HOW-TO-DELETE-MOVE-OR-RENAME-LOCKED-FI
 
 === Create symbolic links
 Are you are used to symbolic and hard links in Linux? Do you have to work with Windows? Would you also like to have such links in Windows? Why not? Windows https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/[supports real links] (not shortcuts like in other cases).
-If you even want to have it integrated in windows explorer you might want to install http://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html[linkshellextension]. However, you might want to disable `SmartMove` in the http://schinagl.priv.at/nt/hardlinkshellext/hardlinkshellext.html#configuration[configuration] if you face strange performance issues when moving folders.
+If you even want to have it integrated in windows explorer you might want to install http://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html[linkshellextension]. However, you might want to disable SmartMove` in the http://schinagl.priv.at/nt/hardlinkshellext/hardlinkshellext.html#configuration[configuration] if you face strange performance issues when moving folders.
```
Comment (ssarmokadam) [inline documentation/advanced-tooling-windows.asciidoc:84]: remove both backticks of SmartMove

File: documentation/features.asciidoc [hunk 10]
```diff
@@ -62,7 +62,7 @@ However, also other IDEs, platforms, or tools can be easily integrated as link:c
 You are not inside a devonfw-ide installation: /
 [/]$ cd /projects/devonfw
 [devonfw]$ mvn
-zsh: command not found: mvn
+`zsh`: command not found: mvn
```
Comment (ssarmokadam) [inline documentation/features.asciidoc:65]: We added this word to dict so remove bacticks

File: documentation/java.asciidoc [hunk 11]
```diff
@@ -22,22 +22,22 @@ Examples for create a new devon4j application:
 ```
 devon java create com.example.domain.myapp
 ```
-Will create an app with package `com.example.domain.myapp`, groupId `com.example.domain`, artifactId `myapp`, version `1.0.0-SNAPSHOT`, and h2 database.
+Will create an app with package `com.example.domain.myapp`, groupId `com.example.domain`, `artifactId` `myapp`, version `1.0.0-SNAPSHOT`, and h2 database.
```
Comment (ssarmokadam) [inline documentation/java.asciidoc:25]: groupId not getting detected as spell mistake here?

File: documentation/java.asciidoc [hunk 12]
```diff
@@ -47,7 +47,7 @@ Will create an app with package `com.example.domain.myapp`, groupId `com.example
 ```
 devon java create com.example.domain.myapp -DdbType=oracle -Dversion=0.0.1 com.example.group -Dbatch=batch
 ```
-Will create an app with package `com.example.domain.myapp`, groupId `com.example.group`, artifactId `myapp`, version `0.0.1`, oracle database, and with a batch module.
+Will create an app with package `com.example.domain.myapp`, groupId `com.example.group`, `artifactId` `myapp`, version `0.0.1`, oracle database, and with a batch module.
```
Comment (ssarmokadam) [inline documentation/java.asciidoc:50]: same as above

**Review by ssarmokadam — CHANGES_REQUESTED** [review]

File: documentation/java.asciidoc (hunk 11 above)
Comment (sujith-mn) [reply documentation/java.asciidoc:25]: groupId  is in the dictionary

**Review by ssarmokadam — APPROVED** [review]

**Pull request merged**